### PR TITLE
Desktop View

### PR DESCRIPTION
**Pull request by ctokoli**
Changes made in this section
-Added the desktop menu
-Made the headline responsive 
-Added grid layout to the project section
-Added grid layout to the skill section 
-Made the contact form responsive
-Added flexbox to the footer to make it responsive

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -42,60 +42,67 @@
...
-                        Hello! I am a software developer! I can help you build a product, feature or website. Take a look at my work. 
-                    </p>
-                    <p> If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
-                    <button class="button">Start Collaboration</button>
+                        <p>
+                            Hello! I am a software developer! I can help you build a product, feature or website. Take a look at my work. 
+                        </p>
+                        <p> If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+                        <button class="button">Start Collaboration</button>
+                </div>
```
Comment (binhussen) [inline index.html:55]: - Kindly add some space between the headline & the start collaboration button and between the headline & project section, add also the dot image to stick the Figma design and have a better UI 👍
- ### Figma Design 👇
Screenshot 2023-05-12 215035

### Your Design 👇
Screenshot 2023-05-12 215216

File: index.html [hunk 2]
```diff
@@ -42,60 +42,67 @@
...
+                            <div class="details-button btn">
                                 <button>
                                     See this project
-                                    <div class="union"></div>
+                                    <div class="union white"></div>
+                                    <div class="union black"></div>
                                 </button>
                             </div>
                         </div>
                     </div>
```
Comment (binhussen) [inline index.html:97]: - Kindly, clear the space between images and keep the size and alignment of the Figma design to have better Ui 👍
### your Design 👇
Screenshot 2023-05-12 214827

### Figma design 👇
Screenshot 2023-05-12 214654

File: index.html [hunk 3]
```diff
@@ -195,97 +208,108 @@ <h2 class="project-name">Project name goes here</h2>
...
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        <button class="button download">Get my CV
+            <div class="icon"></div>
+        </button>
+        </div>
+    </div>
+    <section class="competency">
+        <ul class="grid">
+            <li class="language grid-item">
+                <h3>Languages</h3>
```
Comment (binhussen) [inline index.html:235]: - kindly clear unnecessary spaces above and below the about section to stick to the given design and have a better Ui 👍
### Figma design 👇
Screenshot 2023-05-12 214325

### Your Design 👇
Screenshot 2023-05-12 214524
Screenshot 2023-05-12 214447

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @ctokoli , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- Well Structured Code ✔️
- Kept mobile design ✔️
- Used Grid and Flex ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi @ctokoli, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<div align='center'>

</div>

## Good Highlights✔️

- Requested changes have been implemented

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**